docu/sphinx/source/_static/custom.css

```diff
@@ -0,0 +1,3 @@
+dl.function {
+margin-bottom: 50px;
```

Comment:
I don't like `px` units as it has issues with mobile screens, high resolution screens and so on. It is recommended to use `em` or `rem`. In this case `50px` would be `2,9412em` but you can just say `3em`.